src/apps/Agreement/mock-data.js

```diff
@@ -0,0 +1,158 @@
+// https://gist.github.com/sohkai/7b3385a08ca49ce39935d5b76bf8ef82
+
+const disputableVotingApp = {
+  active: true,
+  appAddress: '0x4f82924c7bd4c0e3b5f0f19273e7f8cdcf5c5609', // this will allow you to find more details from the `apps()` observable
+  collateralToken: {
+    address: '0x3AF6b2f907F0c55F279e0ED65751984E6cdC4a42',
```

Comment:
The `TokenBadge` that is used in the view doesn't populate icons with this token, As discussed, @delfipolito is working on a specific component for displaying amounts nicely for these types of cases.